src/components/Layout/Sidebar/Sidebar.hooks.ts

```diff
@@ -10,16 +9,15 @@ function orderByFileName(files: Map<string, StudioFile>) {
   return orderBy([...files.values()], (s) => s.fileName)
 }
 
-function toFileMap(files: string[]) {
-  return new Map(
-    files.map((fileName) => [fileName, fileFromFileName(fileName)])
-  )
+function toFileMap(files: StudioFile[]) {
+  return new Map(files.map((file) => [file.fileName, file]))
 }
 
 function useFolderContent() {
   const recordings = useStudioUIStore((s) => orderByFileName(s.recordings))
   const generators = useStudioUIStore((s) => orderByFileName(s.generators))
   const scripts = useStudioUIStore((s) => orderByFileName(s.scripts))
+  const data = useStudioUIStore((s) => orderByFileName(s.data))
```

Comment:
Perhaps `data` is too vague?

Reply:
I'm leaning towards a yes, maybe we should be explicit and call it `testData` ?